.github/workflows/invoke-load-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Invoke Load Tests
```

Comment:
We just decided to change the name as `Trigger Load Tests`

Reply:
Fixed in 1f1ce36